src/encoded/static/components/facets/exists.js

```diff
@@ -7,14 +7,16 @@ import FacetRegistry from './registry';
 /**
  * Facet renderer for "exists" facets.
  */
-const ExistsFacet = ({ facet, results, mode, relevantFilters, pathname, queryString }) => (
+const ExistsFacet = ({ facet, results, mode, relevantFilters, pathname, queryString, expandedFacets, setExpandFacets }) => (
     <DefaultExistsFacet
         facet={facet}
         results={results}
         mode={mode}
         relevantFilters={relevantFilters}
         pathname={pathname}
         queryString={queryString}
+        expandedFacets={expandedFacets}
+        setExpandFacets={setExpandFacets}
     />
```

Comment:
`DefaultExistsFacet` doesn’t seem to use these new props, nor does `DefaultDateSelector`.